src/five/customerize/browser.py

```diff
@@ -128,7 +128,7 @@ def templateCodeFromViewName(self, viewname):
         template = self.templateFromViewName(viewname)
         #XXX: we can't do template.read() here because of a bug in
         # Zope 3's ZPT implementation.
-        return open(template.filename, 'rb').read()
+        return open(template.filename, 'rb').read().decode('utf-8')
```

Comment:
I think we want:

    data = open(template.filename, 'rb').read()
    if six.PY2:
        return data
    return data.decode('utf-8')